continuous-delivery/sanity-check-test-pypi.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+CURRENT_TAG_VERSION=$(git describe --abbrev=0)
```

Comment:
Do you want a set -e in here to ensure it dies when something fails?

Reply:
good call.